src/test_reconciler.ts

```diff
@@ -5,13 +5,12 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  *
- * @flow
  */
 
 import path from 'path';
-import type {TestFileAssertionStatus, TestAssertionStatus, TestReconciliationState, Location} from './types';
+import {TestFileAssertionStatus, TestAssertionStatus, TestReconciliationState, Location} from './types';
 
-import type {FormattedAssertionResult, FormattedTestResults} from '../types/TestResult';
+import {FormattedAssertionResult, FormattedTestResults} from '../types/TestResult';
 
 /**
  *  You have a Jest test runner watching for changes, and you have
```

Comment:
Can we get rid of the constructor by writing `fileStatuses: {[key: string]: TestFileAssertionStatus} = {};`?